### PR TITLE
Added icon change on disable

### PR DESCRIPTION
**Pull request by AgilePlaya**
Added support for changing of icons if the extension is disabled.

### CODE REVIEW

**Comment by AgilePlaya** [comment]
@Sankalp7943 Take a look when possible.
Review the changes before closing the PR

File: hydrate/background.js [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
+            path : {
+                16: "icons/icon-gray16.png",
+                32: "icons/icon-gray32.png",
+                48: "icons/icon-gray48.png",
+                128: "icons/icon-gray128.png"
+            }
+        });
         toggle.status = false
-        console.log("Stopped extension")
-        alert("Stopped Hydrate!\nClick again to restart")
```
Comment (Sankalp7943) [inline hydrate/background.js:8]: Why you put a semicolon at the end of some lines?

**Review by Sankalp7943 — COMMENTED** [review]
Why you put a semicolon at the end of some lines?

File: hydrate/background.js (hunk 1 above)
Comment (AgilePlaya) [reply hydrate/background.js:8]: Force of habit, used to adding semicolon to javascript

Comment (AgilePlaya) [reply hydrate/background.js:8]: I'll remove them.

See any other issues?

**Review by Sankalp7943 — APPROVED** [review]

**Pull request merged**